bkit_oauth.py

```diff
@@ -89,29 +99,6 @@ def refresh_token(api_key_refresh, url):
     return auth_token, refresh_token, oauth_response
 
 
-def write_tokens(auth_token, refresh_token, oauth_response):
-    bk_logger.debug('writing tokens')
-    preferences = bpy.context.preferences.addons['blenderkit'].preferences
-    preferences.api_key_timeout = int(time.time() + oauth_response['expires_in'])
-    preferences.api_key_life = oauth_response['expires_in']
-    preferences.login_attempt = False
-    preferences.refresh_in_progress = False
-    preferences.api_key_refresh = refresh_token
-    preferences.api_key = auth_token
-
-    props = utils.get_search_props()
-    if props is not None:
-        props.report = ''
-    reports.add_report('BlenderKit Re-Login success')
-    search.get_profile()
-    ui_props = bpy.context.window_manager.blenderkitUI
-    if ui_props.assetbar_on:
-        ui_props.turn_off = True
-        ui_props.assetbar_on = False
-    search.cleanup_search_results()
-    categories.fetch_categories_thread(auth_token, force = False)
```

Comment:
@vilemduha For what was this line? If I import the categories in the newly created structure, I got circular import error. And it is quite hard to get rid of it. So I am just wondering what function categories.fetch_categories_thread() does, if I can remove it really, or rewrite its functionality in some other way.